### PR TITLE
Build a mobile web application with React and Redux.

### PR DESCRIPTION
**Pull request by marilirulita**
### Build a mobile web application to check a list of metrics (numeric values) making use of React and Redux.
- Create two React components: Home and Details
- The Home component renders a list of companies from NASDAQ stock Exchange and Details renders general data from a specific company
- In the Home, you can search for a company name using the search input.
- There are two routes implemented Home and Details with React Router.
- Connect React and Redux.
- Handle events in a React app.
- Write integration tests with a React testing library.
- Use styles in a React app.
- Use store, actions, and reducers in React.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,56 +1,45 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Metrics Webapp
 
-> Description the project.
+> A mobile web application to check a list of metrics about Financial stock data.
 
+https://user-images.githubusercontent.com/50138343/158038577-1e01372c-7a8c-4bfc-9007-867cb979795c.mp4
```
Comment (arslanbisharat) [inline README.md:7]: - [ ] I cannot seem to hear the sound of the video you recorded. Make sure the video has met all the requirements mentioned below: 

- The student recorded a video presenting the features of the project.
- The maximum length of the video is 5 minutes.
- The video contains a clear and objective description of the project in professional English. The topic and purpose of the project are made clear.
- The video contains a demo of the project, highlighting all the project features.
- The video contains a highlight about a piece of the code the student finds interesting or is proud of making.


> **Note:** It's recommended to use [Loom](https://www.loom.com/) to record to video but You can use any tool.

File: README.md [hunk 2]
```diff
@@ -1,56 +1,45 @@
...
+    `npm install`
+- Run the app\
+    `npm start`
+- Eject the app\
+    `npm run eject`
+- Create build app\
+    `npm run build`
+- Test Linters app\
+    `npm run check-linters`
+
```
Comment (arslanbisharat) [inline README.md]: - [x] I would recommend you to add the instructions on how to run the tests.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by rukundoeric — APPROVED** [review]
Hi @marilirulita,

_Excellent job on this project. Keep it up 🎆_

_Congratulations! 🎉  on the splendid job 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_


Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

------

**Pull request merged**